Ascend.java

```diff
@@ -2,6 +2,19 @@
 // and prints them in increasing order.

```

Comment:
OVERALL:
1. Very good submission. The only concerns regarding logic are in FVCalc.java and TimeFormat.java (the second is not expected) 
2. Styling wise there is room for improvement, specifically with names.